en/restore-from-azblob-using-br.md

```diff
@@ -46,106 +64,165 @@ Before restoring backup data on Azure Blob Storage to TiDB using BR, take the fo
         kubectl create secret generic restore-demo2-tidb-secret --from-literal=password=${password} --namespace=test2
         ```
 
-## Step 2. Restore the backup data to a TiDB cluster
+### Step 2. Restore the backup data to a TiDB cluster
```

Comment:
`Step 2.` -> `Step 2:`

Reply:
Or change `Step 1:` back to `Step 1.` Because there is the same format type in line 143